Formula/p/pixi.rb

```diff
@@ -34,6 +34,10 @@ class Pixi < Formula
   end
 
   def install
+    if build.head?
+      system "git", "fetch", "--tags"
```

Comment:
this is probably not needed

Reply:
In my testing it was 🤔 https://github.com/Homebrew/homebrew-core/pull/195551#discussion_r1816719933